app/javascript/components/analysis/selectors.js

```diff
@@ -172,6 +174,17 @@ export const getLayerEndpoints = createSelector(
   }
 );
 
+export const checkGeostoreSize = createSelector(
+  [selectGeostoreSize, getDataLocation],
+  (areaHa, location) => {
+    if (!['aoi', 'geostore'].includes(location.type)) {
+      return false;
+    }
+
+    return areaHa > 100000000;
```

Comment:
can we reverse the condition? negation conditions tend to break one's brain 🤕 
```
if (isAOI || isGeostore) {
   return areaHa > 100000000;
}

return false;
```

Reply:
Totally correct. I only hope you can forgive me.